doc/changes/latest.inc

```diff
@@ -237,6 +237,9 @@ Bugs
 
 - Fix bug where :func:`mne.io.read_raw_persyst` was lower-casing events it found in the ``.lay`` file (:gh:`9746` by `Adam Li`_)
 
+- Fix bug with :meth:`mne.Epochs.concatenate_epochs` where the concatenated events could end up in non-chronological order. (:gh:`9765` **by new contributor** |Jan Sosulski|_)
```

Comment:
```suggestion
- Fix bug with :meth:`mne.concatenate_epochs` where the concatenated events could end up in non-chronological order. (:gh:`9765` **by new contributor** |Jan Sosulski|_)
```

Reply:
I would just push this, but **new contributor** changes should go at the top of the given changelog group. So can you make this change and move the line up?

Reply:
I moved the entry and changed `:meth:` to `:func:`